src/content/docs/identityserver/reference/options.md

```diff
@@ -340,15 +340,19 @@ Logging related settings, including filters that will remove sensitive values an
 
 * **`AuthorizeRequestSensitiveValuesFilter`**
     
-    Collection of parameter names passed to the authorize endpoint that are considered sensitive and will be excluded from logging. Defaults to `id_token_hint`.
+    Collection of parameter names passed to the authorize endpoint that are considered sensitive and will be redacted in logs. Note that authorization parameters pushed to the Pushed Authorization Request (PAR) endpoint are eventually handled by the authorize request pipeline. This filter should be configured to exclude sensitive values wether or not they are pushed, and usually should be set to the same value as `PushedAuthorizationSensitiveValuesFilter`. Defaults to `client_secret`, `client_assertion`, `id_token_hint`.
```

Comment:
Should we point out that this was changed?
```suggestion
    Collection of parameter names passed to the authorize endpoint that are considered sensitive and will be redacted in logs. Note that authorization parameters pushed to the Pushed Authorization Request (PAR) endpoint are eventually handled by the authorize request pipeline. This filter should be configured to exclude sensitive values wether or not they are pushed, and usually should be set to the same value as `PushedAuthorizationSensitiveValuesFilter`. Defaults to `client_secret`, `client_assertion`, `id_token_hint`. The default value was changed in versions 7.2.2, 7.1.2, and 7.0.9 to include `client_secret` and `client_assertion`. 
```

Reply:
Should we point out that the older default was not good - make explicit that the client_secret (or assertion) would be logged in clear text by default?

Reply:
I don't think it's worth pointing out things that were when the reader is reading the texts of now.

Reply:
That makes sense, we generally don't document that bugs were fixed in particular patch versions. On the other hand, I think we should at least track what has changed in default config values over time.